takwimu/templates/takwimu/_includes/navbar/_index.html

```diff
@@ -99,3 +100,17 @@
     </div>
   </div>
 </nav>
+
+<style>
+  .navbar.fixed-top .dropdown-menu-lg li[type="disc"] {
+	  color:#7ab1f2;
+  }
+  .navbar.fixed-top .dropdown-menu-lg ul li[type="disc"] a{
+      color:#7ab1f2;
+      text-decoration: underline;
+      line-height: 2;
+  }
```

Comment:
@nerdlet Please remove `.navbar.fixed-top` selector. It doesn't do anything not already achieved with `.dropdown-menu-lg` by itself.